crates/types/src/simple_certificate.rs

```diff
@@ -71,11 +71,12 @@ impl<TYPES: NodeType> Threshold<TYPES> for UpgradeThreshold {
 
 /// A certificate which can be created by aggregating many simple votes on the commitment.
 #[derive(Serialize, Deserialize, Eq, Hash, PartialEq, Debug, Clone)]
-pub struct SimpleCertificate<TYPES: NodeType, VOTEABLE: Voteable, THRESHOLD: Threshold<TYPES>> {
+pub struct SimpleCertificate<TYPES: NodeType, VOTE: Vote<TYPES>, THRESHOLD: Threshold<TYPES>> {
     /// The data this certificate is for.  I.e the thing that was voted on to create this Certificate
-    pub data: VOTEABLE,
+    pub data: VOTE::Data,
     /// commitment of all the votes this cert should be signed over
-    pub vote_commitment: Commitment<VOTEABLE>,
+    /// Do not use this field, the commitment needs to be calculated instead
+    vote_commitment: Commitment<VOTE>,
```

Comment:
Can we keep this type the same as it was before, since we don't need it anyway: `Commitment<VOTE::Data>`